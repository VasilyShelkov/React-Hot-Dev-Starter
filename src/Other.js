import React from "react";

export default ({ match }) =>
  <div>
    <h1>Router Some Other Page</h1>
    <ul>
      <li>
        <strong>Pathname</strong>: {match.path}
      </li>
    </ul>
  </div>;
