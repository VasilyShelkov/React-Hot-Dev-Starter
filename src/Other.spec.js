import React from 'react';
import renderer from 'react-test-renderer';
import Other from './Other';

describe('src/Other.js', () => {
  it('renders the Other page', () => {
    expect(renderer.create(<Other />).toJSON())
      .toMatchSnapshot();
  });
});
