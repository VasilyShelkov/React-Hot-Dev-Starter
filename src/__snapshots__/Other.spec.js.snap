// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/Other.js renders the Other page 1`] = `
<div>
  <h1>
    Router Some Other Page
  </h1>
  <ul>
    <li>
      <strong>
        Pathname
      </strong>
      : 
      /somewhereElse
    </li>
  </ul>
</div>
`;
